### ISSUE TITLE
Accessibility Tests

### ISSUE THREAD

**Issue by w-ensink** [privacybydesign/irmamobile#314]
Accessibility tests should be added.
Things like the contrast ratios and hit target sizes can be tested automatically using Flutter.
